docs/InstallEbpf.md

```diff
@@ -11,6 +11,16 @@ Do the following from within the VM:
 1. Download the .msi file from the latest [release](https://github.com/microsoft/ebpf-for-windows/releases)
 2. Execute the .msi file you downloaded
 
+The following components are shown in the MSI to select from:
+
+* Runtime: this is the base eBPF runtime, and is required by the other components.  If you select only this
+  component, only [native code generation](NativeCodeGeneration.md) is enabled.
+* JIT: this adds support for JIT-compiled eBPF programs and (in a Debug build only) interpreted eBPF programs.
```

Comment:
If a user first installs only the `native` mode and then later wants to add the `JIT` mode also, will they need to uninstall eBPF first and then re-install or simply re-running the MSI and choosing JIT option will install the extra binaries / services needed on top of the existing installation?

Reply:
Covered by issue #1276